config/firewalld.service.in

```diff
@@ -2,9 +2,6 @@
 Description=firewalld - dynamic firewall daemon
 Before=network-pre.target
 Wants=network-pre.target
-Requires=dbus.service
```

Comment:
Removing this means firewalld won't be restarted when dbus is restarted. See b9595ea06e6159735300bb4668a3e7e84966219c.

Is there anyway to say "Restart us if dbus restarts, but don't wait on it initially." ?

@mbiebl